app/assets/stylesheets/components/_footer.scss

```diff
@@ -5,7 +5,6 @@
   justify-content: space-between;
   padding: 10px 50px;
   color: white;
-  position: fixed;
```

Comment:
This is something joanna also fixed, but because I couldn't see stuff on my page and use the submit button I deleted it too, there will be small confilcit here when Joanna merges, but we can solve it easily